flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/Row.scala

```diff
@@ -31,6 +31,10 @@ class Row(arity: Int) extends Product {
 
   def setField(i: Int, value: Any): Unit = fields(i) = value
 
+  def getField(i: Int): Any = fields(i)
```

Comment:
Why did you add those methods? `productArity` and `productElement` already do what you want.